doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3070,11 +3146,11 @@ In addition to the known issues listed here, see also :ref:`softdevice_controlle
 
```

Comment:
Here the below known issues can be added:

.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
DRGN-19460: The continuous extended scanner cannot receive the AUX_ADV_IND packet
  This can happen if the time between the ADV_EXT_IND and AUX_ADV_IND was more than 840 μs.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-19580: The stack would dereference a NULL pointer
  This can happen if a resolvable :c:enum:`own_address_type` was used in the HCI Le Extended Create Connection V2 command while the resolving list was empty.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-19623: The HCI Reset would not reset the channel map
  The HCI Reset command would not clear the channel map set by the host using the HCI Le Set Host Channel Classification command.
 
.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-18411: The Peer_Address_Type parameter in the LE Connection Complete event was incorrectly set to 2 or 3 
  This can happen when the connection was established to a device whose address was resolved. The least significant bit of the Peer_Address_Type parameter was set correctly.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-20118: The stack would assert when creating advertisers
  This can happen if trying to set up more advertisers than there are available advertising sets.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0
DRGN-20085: The stack would assert when enabling an advertising set
  Enabling an extended advertising set would assert in cases where a host-provided address was not needed and no address had been set up for the advertising set.

.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-20578: The stack would assert when receiving a non-compliant LL_PHY_RSP
  The controller acting as a central would assert when receiving a non-compliant LL_PHY_RSP from a peer device.